src/pmi/pmi.cs

```diff
@@ -988,6 +1006,29 @@ private List<MethodBase> GetInstances(Type type, MethodBase method)
             }
         }
 
+        // Special case a core async method
```

Comment:
It would perhaps be helpful to expound upon this comment to say why these are particularly interesting to special case..